examples/paillier_logistic_regression.py

```diff
@@ -0,0 +1,227 @@
+"""
+In this example Alice trains a spam classifier on some e-mails dataset she
+owns. She wants to apply it to Bob's personal e-mails, without
+
+1) asking Bob to send his e-mails anywhere
+2) leaking information about the learned model or the dataset she has learned
+from
+3) letting Bob know which of his e-mails are spam or not.
+
+Alice trains a spam classifier with logistic regression on some data she
+posseses. After learning, she generates public/private key pair with a Paillier
+schema. The model is encrypted with the public key. The public key and the
+encrypted model are sent to Bob. Bob applies the encrypted model to his own
+data, obtaining encrypted scores for each e-mail. Bob sends them to Alice.
+Alice decrypts them with the private key and computes the classification error.
```

Comment:
Within your story, Alice cannot compute the classification error, as she has not access to the ground truth.